src/LibVulkan.jl

```diff
@@ -2,29 +2,25 @@ module LibVulkan
 
 import Libdl
 
-@static if Sys.iswindows()
-    const libvulkan = "vulkan-1.dll" 
-elseif Sys.isapple()
-    const libvulkan = "libvulkan.dylib"
-elseif Sys.islinux()
-    const libvulkan = "libvulkan.so.1"
-else
-    const libvulkan = "libvulkan"
```

Comment:
Since swiftshader is only for CI, I think a simpler solution would be to just add a new constant library name for swiftshader like this:

```
@static if Sys.iswindows()
     const libvulkan = get(ENV, "JULIA_VULKAN_CI", "") != "libvk_swiftshader.dll" ? "" : "vulkan-1.dll" 
 elseif Sys.isapple()
     const libvulkan = get(ENV, "JULIA_VULKAN_CI", "") != "libvk_swiftshader.dylib" ? "" : "libvulkan.dylib"
 elseif Sys.islinux()
     const libvulkan = get(ENV, "JULIA_VULKAN_CI", "") != "libvk_swiftshader.so" ? "" : "libvulkan.so.1"
 else
     const libvulkan = get(ENV, "JULIA_VULKAN_CI", "") != "libvk_swiftshader" ? "" :  "libvulkan"
end
```

Reply:
AFAIK the library extensions are not necessary (it is already handled by `dlopen`) unless it is not a standard extension like libvulkan.so.1. However I agree that we should maybe remove `libvk_swiftshader`, we can just set `JULIA_VULKAN_LIBNAME=libvk_swiftshader` as a environment variable instead, since it's just for CI.